pkg/version/version.go

```diff
@@ -16,7 +16,7 @@ var Version = undefinedVersion
 // https://github.com/linkerd/linkerd2-proxy-init
 // This has to be kept in sync with the constraint version for
```

Comment:
This no longer needs to be kept in sync because we're breaking the go dep.  This should be kept in sync with the container image version in values.yml instead.